gearbox.go

```diff
@@ -149,8 +150,9 @@ func (gb *gearbox) newHTTPServer() *fasthttp.Server {
 	}
 }
 
-// Stop serving
+// Print stopping message and stop serving
 func (gb *gearbox) Stop() error {
+	fmt.Printf("Gearbox V%s stopped listening on %s", Version, gb.address)
```

Comment:
What if there is an error is stopping, this message is not in right place.
Check first if shutdown call has an error or on, then print it if there is no error

Reply:
you are right.
got it.